docs/player-config.md

```diff
@@ -129,6 +129,7 @@ const player = Playable.create(config, theme);
 ### Other
 
 `config.videoElement: HTMLVideoElement` You can pass your own HTMLVideoElement to player and we gonna use it.
+**Don't** use `videoElement` attributes to set player options. It will be overwritten by player config.
```

Comment:
Better something like "Attributes from passed videoElement would be override by player"